src/components/input/input.stories.tsx

```diff
@@ -0,0 +1,127 @@
+import type { StoryFn } from '@storybook/react'
+import React from 'react'
+
+import Input from './input'
+import {
+  delayReject,
+  getRandomDelayInSeconds,
+  onFileAddSuccess,
+  onFileDelete,
+} from './mockFunctions'
+
+function onFileAddFailed(
+  file: File,
+  fileId: string,
+  onUploadProgress: (progressEvent: ProgressEvent) => void,
+  abortController: AbortController
+): Promise<{ url: string }> {
+  const deplayTimeInSeconds = 5
+  return delayReject(
+    getRandomDelayInSeconds(deplayTimeInSeconds),
+    abortController.signal
+  )
+}
+
+function onFileAddRandom(
```

Comment:
Should this and `onFileAddFailed` go to `/mockFunctions`?

Reply:
`onFileAddRandom` has been removed. I added `mockFunctions` file because some mock functions are shared by the cypress tests and storybook. I didn't add `onFileAddFailed` to the `mockFunctions` file because I have two different `onFileAddFailed` functions for tests and storybook. I made the cypress one reject quickly so that tests take shorter time. Do you think I should move functions to the `mockFunctions` file even if they are only used in one place?